sos/report/plugins/ceph_mds.py

```diff
@@ -0,0 +1,107 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+from socket import gethostname
+import os
+
+
+class CephMDS(Plugin, RedHatPlugin, UbuntuPlugin):
+
+    short_desc = 'CEPH mds'
+
+    plugin_name = 'ceph_mds'
+    profiles = ('storage', 'virt', 'container')
+
+    containers = ('ceph-fs.*',)
+
+    ceph_hostname = gethostname()
+
+    services = (
+        'ceph-mds@%s' % ceph_hostname,
+    )
```

Comment:
Stylistically this can all be on one line (ditto for the other new plugins).

Reply:
ack thank you I will make these changes

Reply:
ack thank you